includes/editor-templates/panel-elements.php

```diff
@@ -17,20 +17,27 @@
 
 <script type="text/template" id="tmpl-elementor-panel-categories">
 	<div id="elementor-panel-categories"></div>
-
 	<?php
 	$get_pro_details = apply_filters( 'elementor/editor/panel/get_pro_details', [
```

Comment:
Currently I am seeing this panel inside the Pro version as well.

Reply:
@ha
AFAIK, You should see this promotion when Pro is activated.
In Pro it is filtered by a different filter. 
The `free to pro` promotion leads you to `https://elementor.com/pro/?utm_source=panel-widgets&utm_campaign=gopro&utm_medium=wp-dash`3
While in pro, the link leads to `wp-admin/admin.php?page=elementor-apps`

Furthermore, you can see that the promotion text in free and in Pro is different.